README.md

```diff
@@ -144,6 +144,14 @@ Version History
 
 August 2015:
 
+v0.4.2: Stable currency patch.
+- Uploading and Hosting Bugfixes
+- Consensus Bugfixes
+
+v0.4.1: Stable currency patch.
+- Minor Bugfixes
+- Improvements to Build Process
```

Comment:
Sia had a few minor changes as well, mostly with the naming scheme... I think. Maybe not.